salt/metalk8s/orchestrate/upgrade/precheck.sls

```diff
@@ -0,0 +1,17 @@
+{%- set dest_version = pillar.orchestrate.dest_version %}
+
+{%- for node, values in pillar.nodes.items() %}
+
+  {%- if abs(float(dest_version) - float(values['version'])) > 0.1 %}
```

Comment:
Urgh no please don't do this.

Reply:
Why ? How do you want to do ?

Reply:
A version is not a float. Use `pkg_resources.parse_version` or similar.

Reply:
Yes I know but ... ok i will find something better, (`parse_version` does not exist)